### ISSUE TITLE
plotExpression doesn't work properly

### ISSUE THREAD

**Issue by LTLA** [davismcc/archive-scater#131]
Running `example(plotExpression)` gives me a plot where each gene is its own panel, rather than having a single panel containing violin plots for all of the genes. To wit:

```r
## prepare data
data("sc_example_counts")
data("sc_example_cell_info")
example_sce <- SingleCellExperiment(list(counts = sc_example_counts))
sizeFactors(example_sce) <- colSums(counts(example_sce))
example_sce <- normalize(example_sce)
     
## default plot
plotExpression(example_sce, 1:15)
```

**Comment by LTLA** [comment]
Furthermore, this also fails:

```r
c("Gene_0001", "Gene_0002") %in% rownames(example_sce)
## [1] TRUE TRUE
plotExpression(example_sce, "Gene_0001", x="Gene_0002")
## Error in exprs_mat[x, ] : subscript out of bounds
```

**Comment by davismcc** [comment]
Fixed in commit 63b6098776b99fbdc821b7328dc17cab820bffaf
